components/Tabs/src/FontThemer/MDCTabBarFontThemer.h

```diff
@@ -24,10 +24,8 @@
  details on replacement APIs.
  Learn more at docs/theming.md#migration-guide-themers-to-theming-extensions
  */
-@interface MDCTabBarFontThemer : NSObject
-@end
-
-@interface MDCTabBarFontThemer (ToBeDeprecated)
+__deprecated_msg("Please use the theming extension instead.") @interface MDCTabBarFontThemer
```

Comment:
Tip: Consider calling out the name of the Pod/Target.  I guess it's `MDCTabBar+Theming`?

Reply:
Leaving it like this allows it to be Cocoapods/Bazel agnostic though. It's also going to be deleted in the next week or so.